utils/chaos_ai/docker/swagger_api.py

```diff
@@ -54,6 +55,8 @@ def startchaos(self, kubeconfigfile, file_id, params):
                 params['iterations'] = config_params['iterations']
                 params['maxfaults'] = config_params['maxfaults']
         # faults = [f + ':' + p for f in params['faults'].split(',') for p in params['podlabels'].split(',')]
+        if params['podlabels'] is None or params['podlabels'] == '':
```

Comment:
Hi @sandeephans pod labels are not being automatically picked up when not defined. Thoughts?
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/threading.py", line 980, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.9/threading.py", line 917, in run
    self._target(*self._args, **self._kwargs)
  File "/app/swagger_api.py", line 58, in startchaos
    if params['podlabels'] is None or params['podlabels'] == '':
KeyError: 'podlabels'
```